Source/Shaders/GlobeVS.glsl

```diff
@@ -167,4 +172,11 @@ void main()
     v_rayleighColor = atmosColor.rayleigh;
     v_distance = length((czm_modelView3D * vec4(position3DWC, 1.0)).xyz);
 #endif
+
+#ifdef APPLY_MATERIAL
+    vec3 finalNormal = normalize(v_normalMC);
```

Comment:
No need to normalize, it is already done: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Shaders/Builtin/Functions/octDecode.glsl#L24